css/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+    margin: 0;
+}
+
+body {
+    background-color: #205375;
+    font-family: 'Merriweather', serif;
+    font-size: 1rem;
+}
+
+.wrapper {
+    width: 100%;
+    padding-top: 50px;
+    text-align: center;
+}
+
+.box-container {
+    justify-content: center;
+    display: flex;
+    align-items: center !important;
+    margin: auto 0;
+ }
+ 
+.box-one {
+    height: 600px;
+    width: 500px;
+    background: #EDEDED;
+    margin-right: 0;
+    
+}
+
+.box-two {
+    height: 600px;
+    width: 500px; 
+}
+
+img {
+    height: 100%;
+    width: 100%;
+}
+h1 {
+    font-size: 2.8125rem;
+    color: #112B3C;
+    margin: 50px auto 20px;
+    font-family: 'Merriweather', serif;
+}
+p {
+    line-height: 2;
+    font-family: 'Montserrat', sans-serif;
+    text-align: justify;
+    margin: 0 50px ;
+}
+
+/*MEDIA QUERIES*/
+/*tablets*/
+
+@media screen and (max-width: 825px) {
+    .wrapper {
+        width: 100%;
+    }
+   .box-container {
+       text-align: center;
+   } 
+    
+.box-one {
+    height: 400px;
+    width: 300px;
+    margin: 0;
+}
+
+.box-two {
+    height: 400px;
+    width: 300px; 
+    margin-left: 0;
+}
+  
+    h1 {
+        font-size: 1.825rem;
+        margin: 40px auto 20px;
+    }
+    p {
+        line-height: 1.3;
+        font-size: .8rem;
+        margin: 0 20px;
+    }
+    
+}
+@media screen and (max-width: 480px) {
+    
+    .wrapper {
+        padding-top: 20px;
+    }
+        
```

Comment:
Please do not add blank lines or spaces, without a reason. To observe best coding practices